### ISSUE TITLE
Multi DataClass for  GraphQLResolver and throw "java.lang.ArrayIndexOutOfBoundsException: 1"

### ISSUE THREAD

**Issue by mgljava** [graphql-java/graphql-java#1238]
I have an Order class defined as follows：
interface Node {
  id: ID!
}
type Order implements Node {
  orderCode: String
  statusUpdatedRecords: [OrderStatusUpdatedRecord!] #Omitting the OrderStatusUpdatedRecord definition
}
resolver：this is no problem
@Component
public class OrderResolver implements GraphQLResolver<Order> {
  public CompletableFuture<List<OrderStatusUpdatedRecord>> statusUpdatedRecords(Order order, 
  DataFetchingEnvironment environment) {
   
  }
}

and i hava another classes defined as follows:
extend type Order {
  billingAddress: BillingAddress #Omitting the BillingAddress definition
}
but：will trhow java.lang.ArrayIndexOutOfBoundsException
@Service
public class AnotherOrderQueryResolver implements GraphQLResolver<Order> {
  public CompletableFuture<List<OrderStatusUpdatedRecord>> billingAddress(Order order, DataFetchingEnvironment environment) {
    // 
  }
}
In summary：If I add the DataFetchingEnvironment parameter to the method statusQuery, I get an error.If I delete this parameter, then there is no problem.

**Comment by andimarek** [comment]
Hi @mgljava, 

this project doesn't have a `GraphQLResolver`: please open an Issue for the project your are using (graphql-java-annotation?)

Thanks
